packages/office-ui-fabric-react/src/components/Button/_base/Button.base.tsx

```diff
@@ -0,0 +1,265 @@
+import * as React from 'react';
+import {
```

Comment:
No contextual menu in Base button :) :) :)

Reply:
happy days :D

One oddity is that each base button (button/menu/split) have different props, but they all share a single IButtonBaseStyleProps and IButtonBaseStyles. I originally gave each base button its own, but they were all dupes of the original, and really, I want them all to be the same.